popup.js

```diff
@@ -42,4 +20,23 @@ document.addEventListener('DOMContentLoaded', async () => {
         popup.classList.add('hidden');
         popup.classList.remove('popup-fade');
     })
-})
\ No newline at end of file
+}
+
+
+const onSave = () => {
+    const username = document.getElementById("sdarot-auto-username").value;
+    if (username) {
+        chrome.storage.sync.set({sdarotUsername: username});
+    }
+
+    const password = document.getElementById("sdarot-auto-password").value;
```

Comment:
Don't you want to insert it also to locator.js?

Reply:
probably should